content/blog/2021-12-05.md

```diff
@@ -0,0 +1,178 @@
+
+Translations of the XMPP Newsletter will be released here (with some delay):
+
+- The French translation appears at [jabberfr.org](https://news.jabberfr.org/category/newsletter/) and [linuxfr.org](https://linuxfr.org/tags/xmpp/public)
+- The German translation appears at [anoxinon.de](https://anoxinon.de/blog/)
+- The Italian translation appears at [NicFab.it](https://www.nicfab.it/)
+
+Many thanks to the translators and their work! This is a great help to spread the news! Please join them in their work or start over with any another language!
+
+## XSF Announcements
+
+- The XSF members have to elected their new [XSF Board and XSF Council](https://xmpp.org/about/xmpp-standards-foundation/#council)! Congratulations to everyone!
+
+- The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects now! Please apply via [Open Collective](https://opencollective.com/xmpp). For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/).
+
+    - Moreover, the XSF has accepted its first project to the fiscal hosting program! A big welcome to the [MAM Plugin for XMPP.js](https://gitlab.com/deblounge/xmppjs-mam-plugin) (note that this is unaffiliated with the upstream [xmpp.js](https://github.com/xmppjs/xmpp.js) project)! They will be working on adding MAM support to xmpp.js with the eventual goal of adding it to the Matrix Bifrost bridge, allowing XMPP users to fetch history from their favorite Matrix channels. You can [donate to the effort here](https://opencollective.com/bifrost-mam).
+
+- The XSF is planning to participate the Google Summer of Code 2022 (GSoC). If you are interested in participating as a student, mentor or as project in general please [add your ideas and reach out to us](https://wiki.xmpp.org/web/Google_Summer_of_Code_2022)!
+
+- Blog and newsletter pages at [xmpp.org/blog](https://xmpp.org/blog) now support multiple languages. We are happy for volunteers to support translating!
+
+## Events
+
+[XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours) - Also, checkout our new [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)!
+
+[Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): Monthly Meeting of XMPP Enthusiasts in Berlin - always 2nd Wednesday of the month.
+
+## Videos
+
+Sam Whited gave a [talk for the XMPP Office Hours](https://www.youtube.com/watch?v=lprIwxyPY2E) about the XSF's new fiscal hosting service!
+
+Guest talk at Berlin XMPP Meetup: [Diving deep into Briar at the XMPP Meetup Berlin](https://nico.dorfbrunnen.eu/posts/2021/diving-at-xmpp/)
+
+## Articles
+
+[PeerTube Live Chat is using XMPP](https://joinpeertube.org/news#live-plugin-app): Give your instance's audience the ability to chat during live streams!
+
+Using XMPP instead of ActivityPub? Well, [openEngiadina decided to do so](https://inqlab.net/2021-11-12-openengiadina-from-activitypub-to-xmpp.html)!
+
```

Comment:
[XMPP is a solution that allows users to have control over their personal information: our choice Snikket](https://www.nicfab.it/en/xmpp-is-a-solution-that-allows-users-to-have-control-over-their-personal-information-our-choice-snikket/)

Reply:
Thanks, should be added with https://github.com/xsf/xmpp.org/pull/1012/commits/9cc292df02b51f24ba92c4b60a7b6d8e3f3103a9